Userland/Services/SystemServer/main.cpp

```diff
@@ -392,49 +123,46 @@ static ErrorOr<void> prepare_bare_minimum_devtmpfs_directory_structure()
     TRY(Core::System::mkdir("/dev/gpu"sv, 0755));
     TRY(Core::System::mkdir("/dev/pts"sv, 0755));
     TRY(Core::System::mount(-1, "/dev/pts"sv, "devpts"sv, 0));
+
+    mode_t old_mask = umask(0);
+    printf("Changing umask %#o\n", old_mask);
```

Comment:
nit: I know that this is copy-pasta, but since you're already touching it:
```suggestion
```

(I don't think this message ever shows up as the stdout of SystemServer is not hooked up to anything)